src/app/fdctl/config.c

```diff
@@ -1125,6 +1124,28 @@ config_parse( int *      pargc,
   if( FD_UNLIKELY( getgid() != 0 && config->gid != getgid() ) )
     FD_LOG_ERR(( "running as gid %i, but config specifies gid %i", getgid(), config->gid ));
 
+  const char * shmem_path = fd_env_strip_cmdline_cstr( pargc, pargv, "--shmem-path","FD_SHMEM_PATH", NULL );
```

Comment:
I don't think we don't want to support arguments like this from fdctl. You can drop this.